docs/src/index.md

```diff
@@ -131,18 +131,24 @@ struct Package
     buildkite::Bool # does it use Buildkite?
     azure_pipelines::Bool # does it use Azure Pipelines?
     gitlab_pipeline::Bool # does it use Gitlab Pipeline?
-    license_files::Vector{@NamedTuple{license_filename::String, licenses_found::Vector{String}, license_file_percent_covered::Float64}} # a table of all possible license files
+    license_files::Vector{LicenseV1}} # a table of all possible license files
     licenses_in_project::Vector{String} # any licenses in the `license` key of the Project.toml
-    lines_of_code::Vector{@NamedTuple{directory::String, language::Symbol, sublanguage::Union{Nothing, Symbol}, files::Int, code::Int, comments::Int, blanks::Int}} # table of lines of code
-    contributors::Vector{@NamedTuple{login::Union{String,Missing}, id::Union{Int,Missing}, name::Union{String,Missing}, type::String, contributions::Int}} # table of contributor data
-    version::Union{VersionNumber, Nothing} # the version number, if a release was analyzed
+    lines_of_code::Vector{LinesOfCodeV1} # table of lines of code
+    contributors::Vector{ContributorsV1} # table of contributor data
+    version::Union{String, Missing} # the version number, if a release was analyzed
```

Comment:
This is really annoying, but I just couldn't get `VersionNumber` working. The string has all the info we need though, so I guess it's OK. Xref https://github.com/apache/arrow-julia/issues/461